sdk-extensions/jaeger-remote-sampler/src/test/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerTest.java

```diff
@@ -74,7 +72,7 @@ protected void configure(ServerBuilder sb) {
                 protected CompletionStage<byte[]> handleMessage(
                     ServiceRequestContext ctx, byte[] message) {
 
-                  ArmeriaStatusException grpcError = grpcErrors.poll();
+                  ArmeriaStatusException grpcError = grpcErrors.peek();
```

Comment:
The flakes are often because okhttp timesout before processing the error, presumably due to CI slowness / freeziness. I think by always throwing the error every time it polls and awaiting for the log message, the flakiness should hopefully be fixed